extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -1229,9 +1265,14 @@ export class AmpAutocomplete extends AMP.BaseElement {
       case Keys.TAB:
         if (this.areResultsDisplayed_() && this.activeElement_) {
           event.preventDefault();
-          this.selectItem_(this.activeElement_);
+          const selectedValue = this.setInputValue_(this.activeElement_);
+          return this.mutateElement(() => {
+            this.selectItem_(selectedValue);
+          });
         }
-        return Promise.resolve();
+        return this.mutateElement(() => {
```

Comment:
This final return block shouldn't be necessary. If we don't consume the `Tab` event, it will blur the input and hide the suggestions through that event.